ow_regolith/include/RegolithSpawner.h

```diff
@@ -30,7 +30,7 @@ class RegolithSpawner
 
   // spawn the regolith model just above the tip of the scoop and apply a force
   // that keeps it in the scoop during the remainder of scooping operation
-  bool spawnRegolithInScoop(bool with_pushback = true);
+  bool spawnRegolithInScoop(bool with_pushback);
```

Comment:
I suggested to remove argument in its entirety and re-introduce it when it has actual usage in the code but anyway.

Reply:
I see your point about not developing a feature before it is in use, but since it's already there and this is part of an interface and I am inclined to keep it.